import_3dm/read3dm.py

```diff
@@ -0,0 +1,83 @@
+""" MIT License
+
+Copyright (c) 2018 Nathan Letwory , Joel Putnam
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE. """
+
+import os.path
+import bpy
+import rhino3dm as r3d
+from .converters import *
+
+def read_3dm(context, filepath, import_hidden):
+    
+    top_collection_name = os.path.splitext(os.path.basename(filepath))[0]
+    if top_collection_name in context.blend_data.collections.keys():
+        toplayer = context.blend_data.collections[top_collection_name]
+    else:
+        toplayer = context.blend_data.collections.new(name=top_collection_name)
+
+    model = r3d.File3dm.Read(filepath)
+    
+    layerids = {}
+    materials = {}
+
+    handle_materials(context, model, materials)
+
+    handle_layers(context, model, toplayer, layerids, materials )
+        
+    for ob in model.Objects:
+        og=ob.Geometry
+        if og.ObjectType not in RHINO_TYPE_TO_IMPORT: continue
+        exporter = RHINO_TYPE_TO_IMPORT[og.ObjectType]
```

Comment:
I suggest the name  "convert_rhino_object". We aren't exporting, but importing. Conversion is a nice middle road, that you already nicely hinted at with the converters submodule

```suggestion
        convert_rhino_object = RHINO_TYPE_TO_IMPORT[og.ObjectType]
```